_python_workout_code/6.2.py

```diff
@@ -0,0 +1,27 @@
+import operator
+
+def  calc(str):
+    list = str.split()
+    try:
+        return calculate(list[1:], list[0])
+    except:
+        return 'Error: function "' + list[0] + '" doesnt exist'
+
+def  calculate(list, oper):
+    ops = {
+        '+' : operator.add,
+        '-' : operator.sub,
+        '*' : operator.mul,
+        '/' : operator.truediv,
+        '%' : operator.mod,
+        '**' : operator.pow,
+    }
```

Comment:
it's better to say `ops = {...}.get(oper, default_op)` or `ops_dict = {}` if you want to leave it without changes in code. anyway, `the name of a variable should explain what does that var refer to`